ImageKitSample/Program.cs

```diff
@@ -0,0 +1,197 @@
+﻿using Imagekit.Sdk;
+using System;
+
+namespace ImagekitSample
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+
+            //1  
+            ImagekitClient imagekit = new ImagekitClient("TestPublicKey", "TestPrivateKey", "https://api.imagekit.io/");
+            // GetFileListRequest model=new GetFileListRequest();
+            // model.type = "file";
+            // model.limit = 10;
+            // model.skip = 0;
+            //var res= imagekit.GetFileListRequest(model);
+
+
+            // imagekit.Instance.GetFileDetail("62d701678c8b75e43661d66d");
+            // imagekit.PurgeCache("https://ik.imagekit.io/dnggmzz0v/default-image.jpg");
```

Comment:
The sample should have a minimal and full-fledged implementation of all methods with all supported values/properties so that developers can copy and paste this into their applications.

Reply:
Update file example is missing.